Orange/widgets/visualize/owdistributions.py

```diff
@@ -201,26 +201,26 @@ def __init__(self):
                     callback=self._on_set_smoothing, createLabel=False)
         self.l_smoothing_r = gui.widgetLabel(box2, "Precise")
 
-        self.cb_disc_cont = gui.checkBox(
+        gui.checkBox(
```

Comment:
I've done this only in this widget. We can reduce the number of attributes in other widgets in the same way.

It's not just about lint. The benefit of doing this is that we actually reduce the number of names that programmer must cope with: the name of the control (e.g. checkbox) is derivable from the name of the attribute it controls, so no more `cb_`-somethings.

Reply:
+1